packages/data/src/world/mq/jabu_jabu_mq.yml

```diff
@@ -84,22 +200,43 @@
     "MQ Jabu-Jabu Wonder Item Falling Like-Likes Explosion 1": "has_explosives"
     "MQ Jabu-Jabu Wonder Item Falling Like-Likes Explosion 2": "has_explosives"
     "MQ Jabu-Jabu Wonder Item Falling Like-Likes Explosion 3": "has_explosives"
+"Jabu-Jabu Back Forked Paths Right 2":
+  dungeon: JJ
+  exits:
+    "Jabu-Jabu Back Forked Paths": "true" #One-sided collision on the web
```

Comment:
Hmm, the web is not in this room. it is in the forked paths room. Gonna need a buffer region most likely.

Reply:
Added the region.